www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+
+![India's placetype structure](images/placetypes.png "India's placetype structure")
+_(above) This image shows the placetype structure in India. The placetypes circled in red (village, habitations, and urban) are the placetypes that Karmashapes catalogs as `locality` records._
+
+Well, this build wasn't possible until 2022, when five openly-licensed datasets were each available and combined to create Karmashapes; this included three vector datasets and two raster datasets.
+
+Last year, the Indian Ministry of Rural Development ([PMGSY](https://geosadak-pmgsy.nic.in/opendata/)) openly released its ‘Habitations’ dataset. This was the first localities dataset released in India with a clear and friendly license. Geospatial census data from Columbia University’s Socioeconomic Data and Applications Center ([SEDAC](https://sedac.ciesin.columbia.edu/data/set/india-india-village-level-geospatial-socio-econ-1991-2001/data-download)) for all Indian villages helped confirm and verify the locations in the PMGSY dataset. Lastly, a vector dataset from the Geographic Names Server created by the National Geospatial-Intelligence Agency ([NGA](https://geonames.nga.mil/geonames/GNSHome/welcome.html)) was used to cross-reference PMGSY and SEDAC data.
+
+Once a single cleaned and verified vector dataset was created, “unique candidate” urban area polygons were synthesized from two additional global raster datasets: the World Settlement Footprint ([WSFEVO](https://download.geoservice.dlr.de/WSF_EVO/)) Evolution from the German Aerospace Center (DLR) at 10m resolution and a global map of land use/land cover (LULC) derived from the European Space Agency’s (ESA) [Sentinel-2 imagery](https://www.arcgis.com/apps/instant/media/index.html?appid=fc92d38533d440078f17678ebc20e8e2), also at 10m resolution. These raster layers were converted to two additional vector datasets, which were used for an additional cross-reference for the primary vector datasets listed below.
+
+![Pradhan Mantri Gram Sadak Yojana example](images/pmgsy.png "Pradhan Mantri Gram Sadak Yojana example")
+
+**[Pradhan Mantri Gram Sadak Yojana (PMGSY)](https://geosadak-pmgsy.nic.in/opendata/)**
+
+* **Coverage:** 1,084,413 points
+* **License**: CC-BY (equivalent)
+* **Description**: Used for locality points across India. Population data may be gathered from this source.
+* **Data Type**: Vector data.
+
+![National Geospatial Intelligence Agency example](images/nga.png "National Geospatial Intelligence Agency example")
+
+**[National Geospatial Intelligence Agency (NGA)](https://geonames.nga.mil/geonames/GNSHome/welcome.html)**
+
+* **Coverage:** 573,007 points
+* **License**: Public Domain
+* **Description**: These are settlement points, and actually exist in Verrazano today as GeoNames-sourced localities. We may be sourcing new localities from NGA, since the initial GeoNames import occurred in ~2018. We may also get names from this source.
```

Comment:
🛑  Important fix (as NGA is also called Geonames):

`These are settlement points, and often exist in Verrazano today as GeoNames.org-sourced localities. Sometimes the NGA localities may be a little offset from GeoNames.org, and sometimes we're gaining new locality points from NGA. We may also get names from this NGA source. The upstream GeoNames.org project previously imported NGA data which Who's On First then imported via GeoNames.org in ~2018.`

From: 

`These are settlement points, and actually exist in Verrazano today as GeoNames-sourced localities. We may be sourcing new localities from NGA, since the initial GeoNames import occurred in ~2018. We may also get names from this source.`

(This is better, but not amazing English prose.)

Reply:
tweaked slightly, but added.